config/landing_pages.yml

```diff
@@ -152,9 +152,6 @@ shared:
       - Spanish
 
   ### Working patterns
-  flexible-working-jobs-in-schools:
```

Comment:
Moving this down so that part-time-school-jobs is the first landing page slug that matches the criteria working_patterns: "part_time". That is how LandingPage#matching matches search criteria to landing page slug, and it's used by LandingPagesHelper#landing_page_link_or_text to display a link to the landing page with the correct search criteria for part time/full time jobs.